ngprc

```diff
@@ -16,6 +16,16 @@ editor = "vim -c 'set hls' -c 'silent /%1$s' -c %2$d %3$s"
 //editor = "emacs +%2$d %3$s &"
 //editor = "subl %3$s:%2$d 1>/dev/null 2>&1"
 
+/*
+ * external parser commands :
+ *     arg %1$s = options
+ *     arg %2$s = pattern to search
+ *     arg %3$s = directory
+ */
+default_parser = "ag"
```

Comment:
I personally want to keep the native parser as default.

Reply:
Done!
It was already the default if this option would not be set. I changed the default config to make it a little bit more explicit. However the config file handling needs an update, because of the changed interaction with the command line arguments.